### PR TITLE
Updated Introduction Section

### PR DESCRIPTION
**Pull request by corygstevenson**
I added some more context to the introduction on the front page. 

Cleaned up the support page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
+
+## Objectives
+
+1. Decreased Deployment challenges
+2. Increased access to funding
+3. Simplified adoption strategy
+
+OHDSI on Azure is a set of scripts, templates, and DevOps pipelines designed to automate the deployment of the OHDSI in the Microsoft Azure cloud using Terraform & PaaS services. It is designed to facilitate standardized scalable deployments within customer managed Azure subscriptions. Provide best practices for running OHDSI on Azure. Ease the burden of management and cost monitoring of research projects.
+
+OHDSI on Azure has taken a container-based approach to operating OHDSI tools. Therefore, OHDSI on Azure does it’s best to not host code developed by the [OHDSI community](https://github.com/OHDSI). Our deployment templates pull containers from Docker Hub.
```
Comment (plooploops) [inline README.md:15]: Small update, can we change: "Therefore, OHDSI on Azure does it’s best..." to "Therefore, OHDSI on Azure does its best"?

**Comment by plooploops** [comment]
@corygstevenson - I had just a small comment but otherwise looks great!

File: README.md (hunk 1 above)
Comment (corygstevenson) [reply README.md]: Fixed in the latest commit.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,26 @@
...
+
+OHDSI on Azure is a set of scripts, templates, and DevOps pipelines designed to automate the deployment of the OHDSI in the Microsoft Azure cloud using Terraform & PaaS services. It is designed to facilitate standardized scalable deployments within customer managed Azure subscriptions. Provide best practices for running OHDSI on Azure. Ease the burden of management and cost monitoring of research projects.
+
+OHDSI on Azure has taken a container-based approach to operating OHDSI tools. Therefore, OHDSI on Azure does its best to not host code developed by the [OHDSI community](https://github.com/OHDSI). Our deployment templates pull containers from Docker Hub.
+
+We encourage customers to perform their due diligence as part of the pipeline deployments. CI/CD pipelines can be modified to fit your organization’s requirements. Our goal is to get a Sandbox environment setup and provide starter tools for deploying to other environments (DEV/Test, Staging, and Production).
+
+We invite you and your organization to participate in the continued feature expansion of OHDSI on Azure.
+
+This repository assumes the end user is familiar with the OHDSI/ OMOP community, Azure, and Terraform.
```
Comment (plooploops) [inline README.md:21]: Small grammar bit - "OHDSI/ OMOP" should be "OHDSI / OMOP"

Comment (plooploops) [inline README.md:21]: Small grammar bit - "OHDSI/ OMOP" should be "OHDSI / OMOP"

**Review by plooploops — COMMENTED** [review]
Looks good so far, just a small grammar comment!

**Pull request merged**